src/settings-ui/Settings.UI/SettingsXAML/Controls/SettingsPageControl/SettingsPageControl.xaml

```diff
@@ -46,7 +47,10 @@
                 </Grid.RowDefinitions>
 
                 <!--  Top panel  -->
-                <Grid MaxWidth="{StaticResource PageMaxWidth}" ColumnSpacing="16">
+                <Grid
+                    x:Name="TopGrid"
```

Comment:
I think we want to avoid naming things that we don't reference in code-behind. Next time someone wants to edit this code, it feels like I shouldn't touch it because there might be C# referencing this?

Reply:
This was to use with the VisualState. Should it be `Name` instead?